### PR TITLE
fix web administration console message truncate bug APLO-362

### PR DESCRIPTION
**Pull request by shanielh**
https://issues.apache.org/jira/browse/APLO-362

The fix is in the lines 765 - 769, No idea why the diff is so big.

### CODE REVIEW

**Comment by shanielh** [comment]
I've got even a better fix (Change to one line of code).

I'll change the pull request in a few hours.

File: apollo-web/src/main/scala/org/apache/activemq/apollo/web/resources/BrokerResource.scala [hunk 1]
```diff
@@ -760,7 +760,7 @@ class BrokerResource() extends Resource {
     rc.entry = entry
 
     if( max_body > 0 ) {
-      val body = delivery.message.getBodyAs(classOf[Buffer])
+      val body = new Buffer(delivery.message.getBodyAs(classOf[Buffer]))
```
Comment (shanielh) [inline apollo-web/src/main/scala/org/apache/activemq/apollo/web/resources/BrokerResource.scala:763]: non deep clone of the buffer got from the server, It's fast and not mutating the original buffer 
from the server.

**Comment by chirino** [comment]
could you squash your commits?

**Comment by shanielh** [comment]
Yeah. I'll do it tomorrow. 

שני.

**Comment by shanielh** [comment]
Hi, Done it in pull request #9 .

Closing the current one.

**Pull request closed without merging**